precompile/testutils/test_precompile.go

```diff
@@ -85,8 +88,13 @@ func (test PrecompileTest) setup(t testing.TB, module modules.Module, state cont
 	}
 
 	blockContext := contract.NewMockBlockContext(big.NewInt(test.BlockNumber), 0)
-	accesibleState := contract.NewMockAccessibleState(state, blockContext, snow.DefaultContextTest())
-	chainConfig := contract.NewMockChainState(commontype.ValidTestFeeConfig, false)
+	chainConfig := test.ChainConfig
+	if chainConfig == nil {
+		// DUpgrade is activated by default
+		chainConfig = contract.NewMockChainConfig(commontype.ValidTestFeeConfig, false, utils.NewUint64(0))
```

Comment:
kinda nitpicky, but it would be nice to explicitly specify this instead of modifying it here.

Reply:
it would cause panics in precompile tests. added some comments for the default config.